userspace/engine/rule_loader_reader.cpp

```diff
@@ -435,27 +435,28 @@ static void read_item(
 		rule_loader::context ctx(item, rule_loader::context::LIST, name, parent);
 		rule_loader::list_info v(ctx);
 
-		bool append = false;
+		bool has_append_flag = false;
```

Comment:
Why the variable name change? The reason why it was called `append` is because it is true if either the `append` flag is true or there is an append override set (see line 457), hence the wording `append` instead of `has_append_flag`.

Reply:
before my PR `has_append_flag` was already used in the 'rule` case. I adopted it also in the `list` and `macro` case to have consistency, but no strong opinion about the naming

Reply:
There's a difference, in the rule case that variable means that there is an append flag set, as the override append behavior is different (and does not change the flag). I think it's clearer with the `append` name. Thank you :)

Reply:
done, thanks!